config_files/binds/x_touch_mini_binds.yaml

```diff
@@ -0,0 +1,31 @@
+name: XTM_binds
+app_name: napari
+controller_name: X_TOUCH_MINI
+description: xtouchmini_annotation_binds
```

Comment:
The `name` and `description` _can_ contain spaces. Maybe they would look better without `_`?